src/MicrosoftAuthFlow.js

```diff
@@ -134,19 +134,41 @@ class MicrosoftAuthFlow {
     }
   }
 
-  async getMinecraftJavaToken () {
+  /**
+   * @typedef {Object} JavaTokenOptions
```

Comment:
You can put this these in a d.ts file so other programs can actually utilize them

Reply:
I don't know how to implement this, someone will have to do this for me.

Reply:
I've implemented the index.d.ts file, just so you know i hated every minute of it.